calicoctl/commands/ipam.go

```diff
@@ -40,7 +40,7 @@ Description:
 
   See 'calicoctl ipam <command> --help' to read about a specific subcommand.
 `
-	arguments, err := docopt.Parse(doc, args, true, "", false, false)
+	arguments, err := docopt.Parse(doc, args, true, "", true, false)
```

Comment:
Just changing in a couple of places -- it fixes the --help

Without that --help operates on the parent command rather than the sub command.